server/resources/startConfig.yaml

```diff
@@ -24,17 +23,18 @@ workloads:
   hello2:
     runtime: podman
     agent: agent_B
-    restart: true
+    restart: ALWAYS
     tags:
       - key: owner
         value: Ankaios team
     runtimeConfig: |
       image: alpine:latest
-      commandArgs: [ "echo", "Hello Ankaios"]
+      commandOptions: [ "--entrypoint", "/bin/sh" ]
+      commandArgs: [ "-c", "echo 'Always restarted.'; sleep 2"]
   hello-pod:
     runtime: podman-kube
     agent: agent_B
-    restart: true
+    restart: NEVER
```

Comment:
I would not set restart to true for more then one workload inside the startConfig.yaml. It's enough to have one simple workload showing the restart feature. Previously there were a lot of "restart: true" set in documentation and in .yaml files, but the use case was not exactly specified. I have reduced the amount of restart: true stuff. It shall not fail if the restart feature would have some bugs. I prefer only to use restart settings when it is needed in the provided context (because the documentation sets and explains it or there is a technical use case for it in the context).